codex_of_the_damned/templates/archetypes/contributions/creeping-sabotage.html

```diff
@@ -0,0 +1,141 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/contributions/_contributors.html" as contributor with context %}
+{% import "archetypes/contributions/creeping-sabotage.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-nosferatu.svg') }}"> Creeping Sabotage</h1>
```

Comment:
note you can also use the `clan-nosferatu-v5.svg` v5 icon

Reply:
```suggestion
<h1><img src="{{ url_for('static', filename='img/icons/clan-nosferatu-v5.svg') }}"> Creeping Sabotage</h1>
```